src/content/ui/animations/index.md

```diff
@@ -186,10 +164,247 @@ or might partially or completely overlap.
   <img src="/assets/images/docs/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
 {% endcomment -%}
 
+<a id="concepts"></a>
+
+## Essential animation concepts and classes
+
+The animation system in Flutter is based on typed
+[`Animation`][] objects. Widgets can either incorporate
+these animations in their build functions directly by
+reading their current value and listening to their state
+changes or they can use the animations as the basis of
+more elaborate animations that they pass along to
+other widgets.
+
+<a id="animation-class"></a>
+
+### Animation<wbr>\<double>
+
+In Flutter, an `Animation` object knows nothing about what
+is onscreen. An `Animation` is an abstract class that
+understands its current value and its state (completed or dismissed).
+One of the more commonly used animation types is `Animation<double>`.
+
+An `Animation` object sequentially generates
+interpolated numbers between two values over a certain duration.
+The output of an `Animation` object might be linear,
+a curve, a step function, or any other mapping you can devise.
+Depending on how the `Animation` object is controlled,
+it could run in reverse, or even switch directions in the
+middle.
+
+Animations can also interpolate types other than double, such as
+`Animation<Color>` or `Animation<Size>`.
+
+An `Animation` object has state. Its current value is
+always available in the `.value` member.
+
+An `Animation` object knows nothing about rendering or
+`build()` functions.
+
+### Curved&shy;Animation
```

Comment:
This is interesting. Why `&shy;` here? I've never seen `@shy` before, so I googled and I gather it means that the renderer can remove a space, if it wants to. Since this is a class name, the directive seems out of place.

Reply:
I'm honestly not sure why this was here. I also noticed it when copy&pasting. I'll rip it out. There are a few of them.